lib/user-interface/react/src/components/chatbot/Chat.tsx

```diff
@@ -319,41 +325,10 @@ export default function Chat({ sessionId }) {
     const inputs = {
       input: userPrompt,
     };
-    const llm = new Lisa({
-      uri: `${RESTAPI_URI}/${RESTAPI_VERSION}`,
-      idToken: auth.user?.id_token,
-      modelName: selectedModel.modelName,
-      providerName: selectedModel.provider,
-      modelKwargs: modelKwargs,
-      streaming: streamingEnabled,
-      contentHandler: new LisaContentHandler(),
-      callbacks: [
-        {
-          handleLLMNewToken: async (token) => {
-            setSession((prev) => {
-              const lastMessage = prev.history[prev.history.length - 1];
-              const newMessage = new LisaChatMessage({
-                ...lastMessage,
-                content: lastMessage.content + token,
-              });
-              return {
-                ...prev,
-                history: prev.history.slice(0, -1).concat(newMessage),
-              };
-            });
-          },
-        },
-      ],
-    });
-    const llmNoCallback = new Lisa({
-      uri: `${RESTAPI_URI}/${RESTAPI_VERSION}`,
-      idToken: auth.user?.id_token,
-      modelName: selectedModel.modelName,
-      providerName: selectedModel.provider,
-      modelKwargs: modelKwargs,
-      streaming: false,
-      contentHandler: new LisaContentHandler(),
-    });
+
+    const llm = createOpenAiClient(streamingEnabled);
+    const llmNoCallback = createOpenAiClient(false);
```

Comment:
Now that neither of them are using a callback and the logic for updating the chat session data is part of the chain it would make it more clear to rename this to indicate it's used for prompt contextualization. It may be possible to remove it all together too.

Reply:
Good call, I can embed this directly into 'contextualizeQChain'